lib/api/objectDelete.js

```diff
@@ -51,6 +51,13 @@ function objectDeleteInternal(authInfo, request, log, isExpiration, cb) {
     const reqVersionId = decodedVidResult;
     const hasGovernanceBypass = hasGovernanceBypassHeader(request.headers);
 
+    // Reassign a specific apiMethod, as monitoring can rely on the
+    // type of cloudserver instance to determine it comes from Backbeat routes.
+    // This is also needed for quota evaluation, where "routeBackbeat" cannot
+    // be used for different operations, such as putObject
+    // eslint-disable-next-line no-param-reassign
+    request.apiMethod = reqVersionId ? 'objectDeleteVersion' : 'deleteObject';
```

Comment:
instead of the "generic" objectDelete, we do this only in `deleteObjectFromExpiration` :

```suggestion
function deleteObjectFromExpiration(request, response, userInfo, log, callback) {
    // Reassign a specific apiMethod, as it is needed for quota evaluation (at least), where
    // "routeBackbeat" cannot be used as it is used for all backbeat API operations...
    // eslint-disable-next-line no-param-reassign
    request.apiMethod = 'objectDelete';
```